src/test/java/org/HdrHistogram/HistogramTest.java

```diff
@@ -375,7 +375,7 @@ public void testAdd() throws Exception {
     }
 
     @Test
-    public void testSubtract() throws Exception {
+    public void testSubtractAfterAdd() {
```

Comment:
The way this diff is displayed in GitHub is pretty hard to read; it's probably easier to just ignore the diff view and look at these as a sequence of new tests.